cub/test/half.h

```diff
@@ -244,9 +244,14 @@ struct half_t
   }
 
   /// Divide
-  __host__ __device__ __forceinline__ half_t operator/(const half_t& other) const
+  __host__ __device__ __forceinline__ half_t& operator/=(const half_t& other)
   {
-    return half_t(float(*this) / float(other));
+    return *this = half_t(float(*this) / float(other));
+  }
+
+  friend __host__ __device__ __forceinline__ half_t operator/(half_t self, const half_t& other)
+  {
+    return self /= other;
```

Comment:
I fell over the missing `half_t::operator/=` and added it. For consistency, we should probably add the other compound operators as well. Should I address this here, open an issue, or rewrite my code to not use the operator?